dev_scripts/env.py

```diff
@@ -123,8 +121,13 @@
 WORKDIR /home/user
 VOLUME /home/user/dangerzone
 
+# Install Poetry under ~/.local/bin.
+# See https://github.com/freedomofpress/dangerzone/issues/351
+RUN pipx install poetry
+ENV PATH="$PATH:/home/user/.local/bin"
+
 COPY pyproject.toml poetry.lock /home/user/dangerzone/
-RUN cd /home/user/dangerzone && poetry install
+RUN cd /home/user/dangerzone && poetry --no-ansi install
```

Comment:
Isn't the `--no-ansi` thing only needed in debian ones?

Reply:
It's needed for Poetry 1.3, regardless of the OS I think.